polaris.shopify.com/pages/examples/alpha-stack-default.tsx

```diff
@@ -1,20 +1,16 @@
 import React from 'react';
-import {AlphaStack, Badge, Text} from '@shopify/polaris';
+import {AlphaStack} from '@shopify/polaris';
+import {Placeholder} from './ExampleStyleGuideComponents/DefaultBlockComponent/Placeholder';
```

Comment:
To fix build errors, after creating a default export in the component file,
```suggestion
import Placeholder from './ExampleStyleGuideComponents/DefaultBlockComponent/Placeholder';
```